recidiviz/ingest/models/ingest_info.py

```
@@ -335,6 +336,8 @@ def to_repr(obj):
 
 
 def restricted_setattr(self, last_field, name, value):
+    if isinstance(value, str) and (value == '' or value.isspace()):
```

@zacharylawrence we talked about what the converter should do if it gets empty strings from IngestInfo, and had last settled on ignoring the error from `normalize` and converting empty strings to `None`. On second thought I think it might be better handled in IngestInfo. wdyt?

---

Yeah, I like this look! Probably makes sense to also through the exception in the Data Converter though?

---

Cool! It's still handled in converter_utils, I undid that change.